frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunTitle.tsx

```diff
@@ -15,9 +15,9 @@ const PipelineRunTitle: React.FC<PipelineRunTitleProps> = ({ run }) => {
   const { icon, label } = computeRunStatus(run);
 
   return (
-    <Split hasGutter>
-      <SplitItem>{run.name}</SplitItem>
-      <SplitItem>
+    <Split>
+      <SplitItem className="pf-u-text-break-word">
+        <span className="pf-u-pr-md">{run.name}</span>
         <Label icon={icon}>{label}</Label>
       </SplitItem>
```

Comment:
I did this changes to wrap the label with run name.

Reply:
Commented below https://github.com/opendatahub-io/odh-dashboard/pull/1868#discussion_r1343667000

Reply:
cc @yannnz I need some info here. Are we expecting to wrap the label with name or is it okay to display the label side be side like this

![Screenshot from 2023-10-03 14-32-05](https://github.com/opendatahub-io/odh-dashboard/assets/99261071/f8f929e2-8eaa-4ab7-b180-a4fd19d48b0f)

Reply:
> @yannnz I need some info here. Are we expecting to wrap the label with name or is it okay to display the label side be side like this
> 
> ![Screenshot from 2023-10-03 14-32-05](https://user-images.githubusercontent.com/99261071/272206904-f8f929e2-8eaa-4ab7-b180-a4fd19d48b0f.png)

Sorry for chiming in late. Side by side looks good to me.

Reply:
Sorry for chiming in late. Side by side looks good to me.:)